dipy/core/interpolation.pyx

```diff
@@ -64,7 +64,7 @@ def interp_rbf(data, sphere_origin, sphere_target,
     if norm == "angle":
         norm = angle
     elif norm == "euclidean_norm":
-        w_s = "The Eucldian norm used for interpolation is inaccurate "
+        w_s = "The Euclidian norm used for interpolation is inaccurate "
```

Comment:
The correct spelling is "Euclidean" if I'm not wrong.

Reply:
Thanks!